ObjectiveGitFramework.xcodeproj/project.pbxproj

```diff
@@ -1687,15 +1687,15 @@
 				INFOPLIST_FILE = Info.plist;
 				LIBRARY_SEARCH_PATHS = (
 					"$(inherited)",
-					/usr/local/opt/openssl/lib,
+					/opt/homebrew/opt/openssl/lib,
```

Comment:
These all appear to be lists of locations to search. Instead of changing `/usr/local/opt/openssl/lib` to `/opt/homebrew/opt/openssl/lib`, can you just add it (to all of these locations) so that both are present? That way way the linker might be able to build using `/usr/local/opt/openssl/lib` but then load `/opt/homebrew/opt/openssl/lib` @ runtime. Perhaps this could help us get the gitx CI builds to run on x86 and M1?